src/common/mappers/survey-status.mapper.ts

```diff
@@ -13,3 +13,17 @@ export const mapStatusToColor = (status: SurveyStatus): string => {
             return 'volcano';
     }
 };
+
+// See: https://ant.design/components/icon/
+export const mapStatusToIcon = (status: SurveyStatus): string => {
```

Comment:
The return type can be `'check' | 'clock-circle' | 'stop' | 'warning'` for some nice hinting